rotkehlchen/exchanges/manager.py

```diff
@@ -274,3 +277,18 @@ def initialize_exchanges(
                     **extras,
                 )
                 self.connected_exchanges[location].append(exchange_obj)
+
+    def get_all_binance_pairs(self) -> List[str]:
+        if Location.BINANCEUS in self.connected_exchanges:
+            binance = self.connected_exchanges[Location.BINANCEUS][0]
+            return binance._symbols_to_pair.keys()  # type: ignore
+        if Location.BINANCE in self.connected_exchanges:
+            binance = self.connected_exchanges[Location.BINANCE][0]
+            return binance._symbols_to_pair.keys()  # type: ignore
```

Comment:
Prioritize normal binance

```suggestion
        if Location.BINANCE in self.connected_exchanges:
            binance = self.connected_exchanges[Location.BINANCE][0]
            return binance._symbols_to_pair.keys()  # type: ignore
        if Location.BINANCEUS in self.connected_exchanges:
            binance = self.connected_exchanges[Location.BINANCEUS][0]
            return binance._symbols_to_pair.keys()  # type: ignore
```